README.md

```diff
@@ -189,6 +189,76 @@ builder.AddSecretsManager(configurator: options =>
 
 ```
 
+## Configuration via appsettings.json
+Many options can be specified in a JSON file rather than hardcoded within the app. To do so, 
+the `ReadFromConfigSection` method should be called, like so:
+
+```csharp
+builder.AddSecretsManager(configurator: options => 
+{
+    options.ReadFromConfigSection();
+});
+```
+
+By default, this library will assume your options are in a config section named `SecretsManager`, 
+however you can override this behaviour by specifying the name of your config section when calling the method, 
+e.g. `options.ReadFromConfigSection("MySecretsManagerConfig");`
+
+### Example appsettings config section
+The following shows all the options that can be configured via appsettings. Note that each property within this section is optional. 
+```json
+{
+    "SecretsManager": {
+        "Profile": "MySecretsProfile",
+        "Region": "eu-west-1",
+        "ProfilesLocation": "C:\\Users\\MyUser\\.aws\\",
+        "PollingIntervalInSeconds": 60,
+        "AcceptedSecretArns": [
+          "arn:example:01-abcxyz",
+          "arn:example:02",
+          "unique-secret-name"
+        ],
+        "ListSecretsFilters": [
+          {
+            "Key": "Name",
+            "Values": [ "Value1", "Value2" ]
+          }
+        ]
+      }
+}
+```
+
+### Mixing hardcoded and appsettings configuration
+It's possible to configure this library to use options both from code and from a configuration source such as appsettings. 
+To do so, call the `ReadFromConfigSection` method as above, and also specify the configuration that you want to use in code. 
+For example:
+
+```csharp
+builder.AddSecretsManager(configurator: options => 
+{
+    options.ReadFromConfigSection();
+    options.KeyGenerator = (entry, key) => key.ToUpper();
+    options.AcceptedSecretArns = new[]
+    {
+        "MySecretARN1",
+    }
+});
+
+```
+
+Note that if `AcceptedSecretArns` is specified both in code-based and json-based configuration, the collections will be combined
+and both will be used. The same applies for the `ListSecretsFilters` collection. For all other properties, the code-based configuration 
+will take precedence.
```

Comment:
Is it really "code-based precedence will take precedence" or simply "last called wins"?